Hunting Queries/Box/BoxAdminIpAddress.yaml

```diff
@@ -0,0 +1,24 @@
+id: 949aec39-304d-4fba-94b3-15337d05e3f1
+name: Box - IP list for admin users
+description: |
+  'Query shows iplist for admin users. You can check for suspicious IPs or new IPs.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+tactics:
+  - InitialAccess
+  - PrivilegeEscalation
+query: |
+  BoxEvents
+  | where TimeGenerated > ago(60d)

```

Comment:
60d is not recommended, 30d max recommended for hunting query defaults.

Reply:
updated